EIPS/eip-5190.md

```diff
@@ -0,0 +1,59 @@
+
+This EIP also extends account-bound token from [EIP-4973](https://eips.ethereum.org/EIPS/eip-4973), but the difference is that it can be transferred to other account for mobility between managing device.
+
+## Motivation
+
+Finance in crypto ecosystem are facing huge trust issue. Smart contracts can be now a proxy of multisig for operating teams, and Insolvencies do exist in that there is no trace of funds written in blockchain. That can create arbitrary loss of funds as happened in official bankruptcy announcement of Celsius and UST depegging and anchor protocol failure. Smart contract should be used for peer-to-peer only, with immutable rules etched so that only one who has the private key has full access of managing his investment as bitcoin promises. 
+
+Illustration TBD
+
+The pattern empowers more self-sovereignty in users, and gives more credentials(obviously with Account-bound token) than locking financial data in operating team's contract.
+
+Illustration TBD
+
+Segregation between investor's fund and operating fee is clearly specified in the smart contract, so investors can ensure safety to arbitrary loss of funds by operating team's control.
+
+## Specification
+
+The pattern consists of 5 components, manager, factory, finance, account-bound token, extension.
+
+### Diagram
+
+TBD
+
+**`Manager`**: **`Manager`** contract acts as an entry point to interact with the investor. The contract also stores parameters for **`Finance`** contract.
+
+**`Factory`**: **`Factory`** contract manages contract bytecode to create for managing investor's fund and [clones](https://soliditydeveloper.com/clonefactory/) **`Finance`** contract on **`Manager`** contract's approval. It also mints account-bound token to interact with `Finance` contract.
+
+**`Finance`**: **`Finance`** contract specifies all rules on managing investor's fund. The contract is only accessible with account who has Account-bound token. When an investor deposits fund to **`Manager`** contract, the contract sends the fund to **`Finance`** contract after segregation of fees for operation. 
+
+**`Account-bound token`**: **`Account-bound token`** contract in this EIP can bring **`Finance`** contract's data and add in its metadata. For example, if there is a money market lending **`Finance`** contract, its **`Account-bound token`** can show how much balance is in the contract using SVG.
+
+**`Extension`**: **`Extension`** contract is another contract that can utilize locked funds in **`Finance`** contract. The contract is able to access with **`Finance`** contract on operator's approval managed in **`Manager`** contract.
+
+## Reference Implementation
+
+Reference implementation is a CDP-lending stablecoin project [here](https://github.com/digitalnativeinc/standard-evm/tree/master/contracts/vaults/meter).
+
+TBD on specification on each component
+
```

Comment:
```suggestion
```
External links not allowed, and this is an optional section so if you don't have a reference implementation inline or in `../assets/eip-5252/`.